src/namespaces.jl

```diff
@@ -28,7 +31,12 @@ function rimport(pkg::String, s::Symbol=:__anonymous__; normalizenames::Bool=tru
         m = Module(s, false)
         id = Expr(:const, Expr(:(=), :__package__, pkg))
         if normalizenames
-            exports = [Symbol(replace(x, '.' => '_')) for x in members]
+            exports = [Symbol(replace(x, normalization...)) for x in members]
+            if length(exports) > length(unique(exports))
+                dupes = [k for (k, v) in countmap(exports) if v > 1]
```

Comment:
You're identifying unique elements twice; you could `countmap` first and check whether any have a count > 1. Also note that `length(x) > length(unique(x))` can be written more succinctly as `!allunique(x)`.

Reply:
🤦‍♂️